src/Microsoft.SqlTools.SqlCore/Scripting/ScriptAsScriptingOperation.cs

```diff
@@ -53,6 +53,12 @@ public ScriptAsScriptingOperation(ScriptingParams parameters, string azureAccoun
             }
 
             ServerConnection = new ServerConnection(sqlConnection);
+            if (string.IsNullOrEmpty(ServerConnection.Password)) {
+                // Manually add password as server connection class does not add it to string, required for successful connection.
+                if(ServerConnection.ConnectionString.IndexOf("Authentication=SqlPassword") != -1 && ServerConnection.ConnectionString.IndexOf("Password=") == -1) {
```

Comment:
Have you tested the case where the user provides empty password in connection string? i.e. 'Password=;'

Reply:
It works fine, it seems based on stepping through my code that ServerConnection removes the Password from the connection string if it's empty no matter what.